site/assets/js/application.js

```diff
@@ -39,85 +39,79 @@
 
   // Demos within modals
   document.querySelectorAll('.tooltip-test')
-    .forEach(function (tooltip) {
+    .forEach(tooltip => {
       new bootstrap.Tooltip(tooltip)
     })
 
   document.querySelectorAll('.popover-test')
-    .forEach(function (popover) {
+    .forEach(popover => {
       new bootstrap.Popover(popover)
     })
 
   // Indeterminate checkbox example
   document.querySelectorAll('.bd-example-indeterminate [type="checkbox"]')
-    .forEach(function (checkbox) {
+    .forEach(checkbox => {
       checkbox.indeterminate = true
     })
 
   // Disable empty links in docs examples
   document.querySelectorAll('.bd-content [href="#"]')
-    .forEach(function (link) {
-      link.addEventListener('click', function (e) {
+    .forEach(link => {
+      link.addEventListener('click', e => {
         e.preventDefault()
       })
     })
 
   // Modal relatedTarget demo
-  var exampleModal = document.getElementById('exampleModal')
-  if (exampleModal) {
-    exampleModal.addEventListener('show.bs.modal', function (event) {
-      // Button that triggered the modal
-      var button = event.relatedTarget
-      // Extract info from data-* attributes
-      var recipient = button.getAttribute('data-whatever')
-
-      // Update the modal's content.
-      var modalTitle = exampleModal.querySelector('.modal-title')
-      var modalBodyInput = exampleModal.querySelector('.modal-body input')
-
-      modalTitle.textContent = 'New message to ' + recipient
-      modalBodyInput.value = recipient
-    })
-  }
+  const exampleModal = document.getElementById('exampleModal')
+  exampleModal?.addEventListener('show.bs.modal', event => {
```

Comment:
Can you revert the optional chaining changes and revert the .eslintrc.json change? We don't use optional chaining anywhere else right now and I'd like to be consistent.

Reply:
If I remove the eslint file, the linter will complain about any ES6 syntax (i.e. `const`). It's set up to only affect files in the docs folder.

I'll revert the optional chains.

Reply:
NVM about the comment regarding the ESLint changes. Your changes are good since we have an upper eslintrc.json in site folder because the examples still use ES5.